_inc/client/my-plan/index.jsx

```diff
@@ -7,52 +7,52 @@ import { connect } from 'react-redux';
 /**
  * Internal dependencies
  */
-import { getSitePlan, getAvailableFeatures, getActiveFeatures } from 'state/site';
+import { getSitePlan, getSitePurchases, getAvailableFeatures, getActiveFeatures } from 'state/site';
 import QuerySite from 'components/data/query-site';
 import { getSiteConnectionStatus } from 'state/connection';
+import { isJetpackSearch } from 'lib/plans/constants';
 
 import MyPlanHeader from './my-plan-header';
 import MyPlanBody from './my-plan-body';
 
-export class MyPlan extends React.Component {
-	renderContent = () => {
-		let sitePlan = this.props.sitePlan.product_slug || '',
-			availableFeatures = this.props.availableFeatures,
-			activeFeatures = this.props.activeFeatures;
-		if ( 'dev' === this.props.getSiteConnectionStatus( this.props ) ) {
-			sitePlan = 'dev';
-			availableFeatures = {};
-			activeFeatures = {};
-		}
-
-		return (
-			<div>
-				<MyPlanHeader plan={ sitePlan } siteRawUrl={ this.props.siteRawUrl } />
-				<MyPlanBody
-					plan={ sitePlan }
-					availableFeatures={ availableFeatures }
-					activeFeatures={ activeFeatures }
-					siteRawUrl={ this.props.siteRawUrl }
-					siteAdminUrl={ this.props.siteAdminUrl }
-					rewindStatus={ this.props.rewindStatus }
-				/>
-			</div>
-		);
-	};
-
-	render() {
-		return (
-			<div>
-				<QuerySite />
-				{ this.renderContent() }
-			</div>
-		);
+export function MyPlan( props ) {
+	const hasSearchProduct = !! find( props.purchases, purchase =>
```

Comment:
I knew I forgot something; this needs an `import { find } from 'lodash';` for it to work properly.